lib/storage-client.js

```diff
@@ -296,11 +296,10 @@ class ChromedriverStorageClient {
       }
     }
 
-    if (_.isString(minBrowserVersion) && !Number.isNaN(minBrowserVersion)) {
+    if (minBrowserVersion && !Number.isNaN(minBrowserVersion)) {
```

Comment:
this condition could be changed to `/^\d+$/.test(minBrowserVersion)`

Reply:
I think we need a typecast and null check like this https://github.com/appium/appium-chromedriver/pull/291/commits/543d94868bd46836df9a1554b0896e8dc0ad1b08 for typescript linter.